### PR TITLE
Core metric changes

### PR DESCRIPTION
**Pull request by kris-gomes**
Added in the columns for PAR and Maximet start times to core metrics table. Data with start times for these tables should come in after the 6pm data pull from the buoys.

### CODE REVIEW

File: app/routes/telemetry/index.js [hunk 1]
```diff
@@ -47,9 +49,19 @@ const CORE_METRICS = {
...
+  Hydrocycle: [
+  "CAPO4",
+  "QCflag",
+  "Bubbleflag",
+  "COVflag",
+  "Lowsigflag",
+  "OoRflag",
+  "Mixingflag",
+  "Calflag",
+  "Hydrocyclestart"]
```
Comment (mcmcgrath13) [inline app/routes/telemetry/index.js:62]: should this one be `hydrocycleStart`?

**Comment by mcmcgrath13** [comment]
Instead of adding the `start` columns to each table, I would add it to the `router.get` for core metrics
```js
router.get(
  "/:buoyId/CoreMetrics/:range",
  ash(async (req, res) => {
    const results = await Promise.all(
      Object.entries(CORE_METRICS).map(async ([table, vars]) => ({
        [table]: await req.queryFn(req.params.buoyId, table, [
          "TmStamp",
          `${table.toLowerCase()}Start`,
          ...vars,
        ]),
      }))
    );
    // merge the results into one object
    const result = results.reduce((a, b) => ({ ...a, ...b }), {});
    res.send(result);
  })
);
```

**Comment by kris-gomes** [comment]
Gotcha, that's much more straightforward. Updated to include router.get code and removed the individual start times in the tables.

File: app/routes/telemetry/index.js [hunk 2]
```diff
@@ -38,17 +38,26 @@ const RANGES = {
   },
 };
 const CORE_METRICS = {
-  ECO: ["ecoReadingRaw", "ecoFDOM"],
+  ECO: ["ecoReadingRaw",
+   "ecoFDOM",
+   "ecoStart"],
```
Comment (mcmcgrath13) [inline app/routes/telemetry/index.js:43]:
```suggestion
   ],
```

**Review by mcmcgrath13 — COMMENTED** [review]
awesome! could you run `npm run prettify`? - some of the styling looks a little off

**Comment by kris-gomes** [comment]
Sure thing, index.js has been prettified

**Comment by mcmcgrath13** [comment]
Just deployed this change - things look good on my front: https://api.riddc.brown.edu/telemetry/Buoy-720/CoreMetrics/lastday

**Pull request merged**